tv-shows/src/app/all-shows/[id]/page.tsx

```diff
@@ -0,0 +1,7 @@
+'use client';
```

Comment:
I know we said write 'use client' everywhere, but you are a smart boy, you don't need it here as the page has no interactive elements, no handlers, and no hooks.